src/interfaces/utils-interfaces/address-schema-interface.ts

```diff
@@ -0,0 +1,9 @@
+export interface AddressSchemaIF {
+  streetAddress: {}
+  streetAddressAdditional: {}
+  addressCity: {}
+  addressCountry: {}
+  addressRegion: {}
+  postalCode: {}
+  deliveryInstructions: {}
+}
```

Comment:
In Create UI, this file is in a new folder (schemas).

Will you add validation rules here, as in the file below?

https://github.com/bcgov/business-create-ui/blob/main/src/schemas/office-address-schema.ts

Reply:
There is a schema folder in this UI as well which currently houses these :)